client/styles/components/_preferences.scss

```diff
@@ -40,12 +45,31 @@
 	display: flex;
 	flex-direction: row;
 	justify-content: space-between;
-	margin-bottom: #{20 / $base-font-size}rem;
+	marhin-left: #{19 / $base-font-size}rem;
 	@include themify() {
 		color: getThemifyVariable('heading-text-color');
 	}
 }
 
+.preferences__title{
+	width: #{90 / $base-font-size}rem;
```

Comment:
looks like there are some inconsistencies with indentations—mind cleaning this up?